v1.md

```diff
@@ -11,12 +11,19 @@ All of our APIs are designed to allow permissioned users to look up, modify, or
 
 The following key points and conventions are followed:
 
+<div class="alert alert-info">
+  <strong>Root URLs:</strong><p>The root for all API calls is <strong>https://DOMAIN.webadmit.org</strong>.
+  DOMAIN is your organization domain name. The default domain is <strong>api</strong> (https://api.webadmit.org)</p>
```

Comment:
I would maybe write "The default DOMAIN is" instead of "The default domain is" for some clarity. Not a big deal.